superbench/benchmarks/base.py

```diff
@@ -162,24 +162,30 @@ def __check_result_type(self):
 
         return True
 
+    def __is_list_type(self, data, type):
```

Comment:
`type` is builtin function so may be not a good var name

Reply:
OK